amd64/root/firmware-update.sh

```diff
@@ -138,8 +139,11 @@ ls -1 "$FW_PATH"
 echo " "
 echo "Enter the firmware file name from above, including extension."
 echo "Alternatively, you may enter the name of a firmware file to download"
-echo "from $FW_ONLINE_BASE"
-echo "or press Enter now to exit."
+echo "from any of the following sources:"
+echo " - $FW_ONLINE_STABLE_BASE        (Stable)"
```

Comment:
As the script runs with `bash` you might want to use `printf` to get the alignment.

Reply:
```shell
for base in "${FW_ONLINE_BASE_ORDER[@]}"; do
    printf " - %-60s (%s)" "${FW_ONLINE_BASES[$base]}" "$base"
done
```